servers.json

```diff
@@ -1278,5 +1278,12 @@
         "addresses": [
             "mythicalmc.org"
         ]
+     },
+    {
+        "id": "freshpvp",
```

Comment:
Add your server logo to the `logos` directory

Reply:
![Fresh-Mc-logo-2](https://user-images.githubusercontent.com/88015961/164133827-289c7360-d458-4cb9-8cbf-a89a833da9a9.png)
my logo, not working uploaded

Reply:
![pixlr-bg-result (1)](https://user-images.githubusercontent.com/88015961/164134381-b2e0d7f7-5914-46c5-be26-1f33098798ec.png)
not working, no perms uploaded in logos

Reply:
ready, logo freshpvp uploaded